scripts/sanity_chk/sanitylib.py

```diff
@@ -708,6 +708,15 @@ def __init__(self, instance, type_str):
 
         self.pid_fn = os.path.join(instance.build_dir, "qemu.pid")
 
+    @staticmethod
```

Comment:
Please add a comment why you need that. you should consider using psutil hear instead of the custom mixed scripting.

Reply:
@nashif update

Reply:
@nashif CI environment doesn’t have psutil installed which caused shippable Timeout, is it possible setup psutil in CI in short time, though it’s easy on my PC?

Reply:
@wentongwu why haven't you added psutil to scripts/requirements.txt?

Reply:
@wentongwu  for now, just install it directly in shippable.yml, we will do the update soon.

Reply:
@nashif @andrewboie as indicated by https://github.com/giampaolo/psutil/blob/master/INSTALL.rst#linux, psutil depends on python3-dev on Ubuntu / Debian or python3-devel on RedHat / CentOS, which OS our CI is running?